config/config/common-config.yaml

```diff
@@ -381,20 +381,9 @@ prometheus:
     storage:
       predictLinear:
         - hours: 24
-          freeSpacePercentage: 30
+          freeSpacePercentage: 5
```

Comment:
Could this be to low of a value? :thinking: 
But then again, we have the other alerts for actual low space.

Reply:
Perhaps. Suggestions? 20%? Leave as is? AIUI these are "This much free space left and predicted reach 0%".

Reply:
Since we rely on other alerts for low space, I would say leave this one at 5%